peer-verification/README.md

```diff
@@ -0,0 +1,58 @@
+# Peer Verification
+This tool is intended to verify a potential candidate for testnet validator.
+Package is split into two parts: `signer` and `verifier`.
+
+
+Signer part will be used by the candidate.
+With this program candidate is going to sign a secret message provided by us to prove they own the p2p secret they claim to own.
+
+
+Verifier part will be used by Aleph Zero team.
+With this program we are going to check whether supplied public key derives to supplied peer id, signature of secret message is correct and this peer is connected and up to date with our network.
+
+## Step one: signature generation by Validator candidate
+To generate signature, candidate needs a generated p2p secret.
+For whole process to work they also need to have a node set up with up to date block creation and public p2p port, so it can be added as a peer.
```

Comment:
```suggestion
For whole process to work they also need to have a node set up with up to date block creation and public p2p port, so it can be discovered by other nodes as a peer.
```